config.h

```diff
@@ -529,6 +529,14 @@ Set to \ref On or 1 to enable experimental support for parameters.
 #define NGC_PARAMETERS_ENABLE On
 #endif
 
+/*! \def STRING_REGISTERS_ENABLE
+\brief
+Set to \ref On or 1 to enable experimental support for string registers.
+*/
+#if !defined STRING_REGISTERS_ENABLE || defined __DOXYGEN__
+#define STRING_REGISTERS_ENABLE On
+#endif
+
```

Comment:
I must admit I am not 100% sure I understand how this works, but it did seem to work with PIO atleast 😅

Reply:
Have read up a bit, and now see that this makes it enabled by default, so I should change this 😅